spec/src/main/asciidoc/core/packagingdeployment.asciidoc

```diff
@@ -16,7 +18,7 @@ Portable extensions may even integrate with the process of building the `Bean` o
 
 [[bean_archive]]
 
-=== Bean archives
+=== Bean archives in {cdi_full}
```

Comment:
Ah true! I added the `_lite` file for this, because that's something we want to write from scratch. Not sure it was the best idea, now that I'm looking at section IDs. Let's think about that -- doesn't have to be solved in this PR for sure :-)